### PR TITLE
// INSTALLER : add PHP memory limit recommendation

### PR DESCRIPTION
**Pull request by kelu95**

### CODE REVIEW

File: classes/ConfigurationTest.php [hunk 1]
```diff
@@ -363,11 +364,21 @@ public static function test_sessions()
...
+
     public static function test_dom()
     {
         return extension_loaded('Dom');
     }
 
+    public static function test_php_memory_limit()
+    {
+        // Minimum memory limit for Windows : 1024, else 512
+        $recommended_memory_limit = strtoupper(substr(php_uname('s'), 0, 3)) === 'WIN' ? 1024 : 512;
```
Comment (mickaelandrieu) [inline classes/ConfigurationTest.php]: It's far too much!

Comment (xGouley) [inline classes/ConfigurationTest.php]: Nope, it's needed to list products on catalog page. Tested.

Comment (mickaelandrieu) [inline classes/ConfigurationTest.php]: if we force this behavior, we lose mostly 99% of our shops imo :)

**Comment by Quetzacoalt91** [comment]
We planned to remove these performance issues, right ?

If we add these requirements and fix this restriction, I'm pretty sure we will forget to remove them later. If I'm wrong, this PR will be reopened, but currently we do not need to merge this PR.

Best regards

**Pull request closed without merging**